src/makeRobot.js

```diff
@@ -38,7 +38,72 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: `${name}`,
+    wheels: `${wheels}`,
+    version: `${version}`,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(coordinate = 1) {
+      if (typeof coordinate !== 'number' || coordinate < 0) {
+        return this;
+      }
+
+      this.coords.y += coordinate;
+
+      return this;
+    },
```

Comment:
Don't repeat yourself 

```suggestion
    goForward(coordinate = 1) {
      if (coordinate >= 0) {
        this.coords.y += coordinate;
      }

      return this;
    },
```